phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolTimeRangeIT.java

```diff
@@ -226,7 +225,8 @@ public void tickTime() {
     }
 
     @AfterClass
-    public static synchronized void teardown() {
+    public static synchronized void teardown() throws Exception {
+        confirmStoreRefCountLeak();
```

Comment:
this should also be
```
try {
  confirmStore...
} catch (Exception e) {
  Assert.fail()}
} finally {
  tearDownMiniCluster...
}
```

Reply:
Oh btw we don't need to worry about putting anything in finally after calling BaseTest#confirmStoreRefCountLeak() because BaseTest is taking care of shutting down miniCluster:
```
    protected synchronized static void confirmStoreRefCountLeak()
            throws Exception {
        if (getUtility() != null) {
            try {
                CompatUtil.confirmStoreRefCountLeak(
                    getUtility().getAdmin());
            } catch (IOException e) {
                LOGGER.error("StoreFile refCount is leaked", e);
                freeResources(false);
                fail("StoreFile refCount is leaked");
            }
        }
    }
```
freeResources() is doing the job.